README.md

```diff
@@ -19,7 +19,7 @@ List of available configurations:
 | ``identifierAttribute`` | | String | Attribute from the profile of the provider to use as Id if you want to persist the user in Kuzzle |
 | ``defaultProfile`` | ``"default"`` | Array | Profiles of the new persisted user |
 | ``kuzzleAttributesMapping`` | ```` | Object | Mapping of attributes to persist in the user persisted in Kuzzle |
-| ``passportStrategy`` | String | Strategy name for passport (eg. google-oauth20 while the name of the provider is google)
+| ``passportStrategy`` | ```` | String | Strategy name for passport (eg. google-oauth20 while the name of the provider is google)
 
 Here is an example of a configuration:
```

Comment:
Maybe you can me a little bit more explicit in your example, something like
`"userMail": "email" // will store the attribute "email" as "userEmail" into Kuzzle`